src/plone/restapi/tests/test_deserialize_siteroot.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
There already is an existing [`src/plone/restapi/tests/test_site_deserializer.py`](https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/tests/test_site_deserializer.py) - can you please integrate your testing changes into that one instead? (And also rename it to something like `TestSiteRootDeserializer` - the Plone site is not DX content).

Reply:
I moved the tests into test_site_deserializer.py and renamed the class name. Thx for the pointer!